modules/logging-loki-storage-odf.adoc

```diff
@@ -3,15 +3,13 @@
 //
 :_mod-docs-content-type: PROCEDURE
 [id="logging-loki-storage-odf_{context}"]
-
-= ODF storage
+= OpenShift Data Foundation storage
```

Comment:
Procedure titles should begin with a gerund, such as "Configuring OpenShift Data Foundation storage".

Reply:
Also, be sure to use the attribute for OpenShift Data Foundation: `rh-storage`